### ISSUE TITLE
Barra de rolagem no navegador web

### ISSUE THREAD

**Issue by walterowisk** [birobirobiro/DevNews#5]
Fala Biro³. O projeto tá ótimo!!
Eu observei algo curioso aqui. A página não está mostrando a barra de rolagem lateral e com isso os 4 cards de notícias ficaram cortados na parte debaixo e consequentemente não aparece o rodapé.

Testei no Linux _Pop_OS 22.04_ nos navegadores _Google Chrome Versão 111.0.5563.110 64 bits_ e no _Firefox 111.0 64-bit_.

**Comment by birobirobiro** [comment]
> Fala Biro³. O projeto tá ótimo!! Eu observei algo curioso aqui. A página não está mostrando a barra de rolagem lateral e com isso os 4 cards de notícias ficaram cortados na parte debaixo e consequentemente não aparece o rodapé.
> 
> Testei no Linux _Pop_OS 22.04_ nos navegadores _Google Chrome Versão 111.0.5563.110 64 bits_ e no _Firefox 111.0 64-bit_.

Fala @walterowisk , valeu pelo feedback.

Qual a resolução da sua tela? Por padrão as barras de rolagem ficam ocultas mesmo, e aparecem em resoluções a partir de 2500px de largura.

**Comment by walterowisk** [comment]
@birobirobiro, acho que foquei no problema errado. A questão da barra de rolagem é consequência. 
O que tá ocorrendo é que pra mim a página simplesmente tá cortanto antes de terminar os cards e com isso não aparece nem o rodapé nem com o F11 ativado. Veja a imagem abaixo.

Se for um problema só meu, ok. Mas de qualquer forma achei por bem te reportar.

![image](https://user-images.githubusercontent.com/57921555/229171254-fcaed4b2-9041-4f6e-a93b-466295075116.png)

**Comment by birobirobiro** [comment]
> 1366x768
Acabei de subir um commit que deve resolver isso.

https://github.com/birobirobiro/DevNews/commit/3ee2fab49d307f19ba984db9f18a1eae9a5d2342

**Comment by matheralvs** [comment]
Opa @birobirobiro, blz? Cara, ia abrir uma issue com esse mesmo problema, mas acabei vendo que já tinha essa daqui aberta. Pra resolução 1920x1080 tá com esse mesmo probleminha. Aliás, que projeto top, tu é fera demais!

**Comment by matheralvs** [comment]
> Opa @birobirobiro, blz? Cara, ia abrir uma issue com esse mesmo problema, mas acabei vendo que já tinha essa daqui aberta. Pra resolução 1920x1080 tá com esse mesmo probleminha. Aliás, que projeto top, tu é fera demais!

Dando uma olhada aqui no projeto com mais calma, foi apenas um ``overflow: hidden`` que foi colocado no body acidentalmente.

![image](https://user-images.githubusercontent.com/109676973/232348090-a7a56921-8798-4b24-8afd-3ac4a0b0b75d.png)

**Comment by birobirobiro** [comment]
> > Opa @birobirobiro, blz? Cara, ia abrir uma issue com esse mesmo problema, mas acabei vendo que já tinha essa daqui aberta. Pra resolução 1920x1080 tá com esse mesmo probleminha. Aliás, que projeto top, tu é fera demais!
> 
> Dando uma olhada aqui no projeto com mais calma, foi apenas um `overflow: hidden` que foi colocado no body acidentalmente. Realizei um PR (#6) que corrige este pequeno problema.
> 
> ![image](https://user-images.githubusercontent.com/109676973/232348090-a7a56921-8798-4b24-8afd-3ac4a0b0b75d.png)

Sim, a ideia era deixar desabilitado mesmo para não ter a rolagem e fazer o ajuste nos breakpoints conforme as resoluções. Mas acho que essa vai ser a melhor solução mesmo, aprovei a sua PR.

Valeu! 🚀
